set.mm

```diff
@@ -469438,6 +469469,445 @@ Splicing words (substring replacement)
+      XSXJYDYOWOXPWRXRYOWNXOWKYOWLDWMCUGYLYNVEZYIYKYNVGZVFVHYOWQXQWPYOWMCWLDUGY
+      QYPVFVHVIYOXIYCAWMCYQYOXHYBBWLDYPYOXDXTXGYAYOXCLXAXBYOXCIUITLYOWIIUIYLYHY
+      NYMVJVBPVCVKYOXFGWTXEYOXFHUITGYOWGHUIYIYGYKYNYJVLVBOVCVKVMVNVNVIVQVRVRWFH
+      JWDWEVOVPWFIKWDWEVEVPWFYEEFXOXQRRWFDCUGVSWFCDUGVSWFXPXRYDVTWFXPXRYDWAWBWC
+      $.
+
+    $d .<_ f g $.  $d .c_ f g $.  $d A f g x y $.  $d B f g x y $.
+    $d F f g x y $.  $d G f g x y $.  $d V f g x y $.  $d W f g x y $.
+    $d f g ph $.
+    mgcval.1 $e |- H = ( V MGalConn W ) $.
+    mgcval.2 $e |- ( ph -> V e. Proset ) $.
+    mgcval.3 $e |- ( ph -> W e. Proset ) $.
+    $( Monotone Galois connection between two functions ` F ` and ` G ` .  If
+       this relation is satisfied, ` F ` is called the lower adjoint of ` G ` ,
+       and ` G ` is called the upper adjoint of ` F ` .  (Contributed by
+       Thierry Arnoux, 23-Apr-2024.) $)
+    mgcval $p |- ( ph -> ( F H G <-> ( ( F : A --> B /\ G : B --> A ) /\ A. x
+       e. A A. y e. B ( ( F ` x ) .c_ y <-> x .<_ ( G ` y ) ) ) ) ) $=
+      ( wa vf vg wbr cv cmap co wcel wb wral copab wf cmgc cproset wceq mgcoval
+      cfv syl2anc syl5eq breqd fveq1 adantr breq1d adantl bibi12d 2ralbidv eqid
+      breq2d brab2a cbs fvexi elmap anbi12i anbi1i bitr2i syl6bbr ) AFGHUCFGUAU
+      DZEDUEUFZUGUBUDZDEUEUFZUGTBUDZVPUPZCUDZLUCZVTWBVRUPZIUCZUHZCEUIBDUIZTUAUB
+      UJZUCZDEFUKZEDGUKZTZVTFUPZWBLUCZVTWBGUPZIUCZUHZCEUIBDUIZTZAHWHFGAHJKULUFZ
+      WHQAJUMUGKUMUGWTWHUNRSBCDEUAUBIJKUMUMLMNOPUOUQURUSWIFVQUGZGVSUGZTZWRTWSWG
+      WRUAUBFGVQVSWHVPFUNZVRGUNZTZWFWQBCDEXFWCWNWEWPXFWAWMWBLXDWAWMUNXEVTVPFUTV
+      AVBXFWDWOVTIXEWDWOUNXDWBVRGUTVCVGVDVEWHVFVHXCWLWRXAWJXBWKEDFEKVINVJZDJVIM
+      VJZVKDEGXHXGVKVLVMVNVO $.
+
+    ${
+      $d A x y $.  $d B x y $.  $d F x y $.  $d G x y $.  $d V x y $.
+      $d W x y $.
+      mgccole.1 $e |- ( ph -> F H G ) $.
+      $( The lower adjoint ` F ` of a Galois connection is a function.
+         (Contributed by Thierry Arnoux, 24-Apr-2024.) $)
+      mgcf1 $p |- ( ph -> F : A --> B ) $=
+        ( vx vy wf cv cfv wbr wb wral wa mgcval mpbid simplld ) ABCDUAZCBEUAZSU
+        BZDUCTUBZJUDUMUNEUCGUDUETCUFSBUFZADEFUDUKULUGUOUGRASTBCDEFGHIJKLMNOPQUH
+        UIUJ $.
+
+      $( The upper adjoint ` G ` of a Galois connection is a function.
+         (Contributed by Thierry Arnoux, 24-Apr-2024.) $)
+      mgcf2 $p |- ( ph -> G : B --> A ) $=
+        ( vx vy wf cv cfv wbr wb wral wa mgcval mpbid simplrd ) ABCDUAZCBEUAZSU
+        BZDUCTUBZJUDUMUNEUCGUDUETCUFSBUFZADEFUDUKULUGUOUGRASTBCDEFGHIJKLMNOPQUH
+        UIUJ $.
+
+      ${
+        $d .<_ x y $.  $d .c_ x y $.  $d A x y $.  $d B x y $.  $d F x y $.
+        $d G x y $.  $d V x y $.  $d W x y $.  $d X x y $.  $d ph x y $.
+        mgccole1.2 $e |- ( ph -> X e. A ) $.
+        $( An inequality for the kernel operator ` F o. G ` (Contributed by
+           Thierry Arnoux, 26-Apr-2024.) $)
+        mgccole1 $p |- ( ph -> X .<_ ( G ` ( F ` X ) ) ) $=
```

Comment:
``( G ` ( F ` X ) )  = ( ( G o. F ) ` X )`` , so the comment is wrong.

Reply:
Yes, thanks!
And the comment for `~mgccole2` is wrong too because `F o. G` is called the closure operator.